### ISSUE TITLE
Disk or RAM

### ISSUE THREAD

**Issue by Nadian-Ali** [LeeDoYup/bottom-up-attention-tf#2]
Hi.. i am trying to run your code... i have 32 GB of RAM and more than 100GB of disk space. 3 GB of my RAM gets occupied for loading the dictionary and i get stuck there.... do i need a bigger RAM? Like 64GB of RAM space?

**Comment by LeeDoYup** [comment]
I remember 
VQA implementations require 150 - 200 GB RAM (also in [pytorch implementation](https://github.com/hengyuan-hu/bottom-up-attention-vqa))

It is because the implementations use pretrained features and load at once..
If you want to save, find or implement a data loader that satisfy
1) Load an image file at each time
2) Detect objects in image
3) and run VQA models.

The reason that many implementations use pretrained features (including this repo) is to save inference time during 1)-2) .

**Comment by LeeDoYup** [comment]
https://github.com/LeeDoYup/bottom-up-attention-tf/blob/master/README.md#prerequisites

**Comment by Nadian-Ali** [comment]
Thank you very much... i am trying to use virtual RAM...  I hope it works

**Comment by LeeDoYup** [comment]
@Nadian-Ali Okay. FYI, i achieve the VQA accuracy similar to pytorch implementation.
I implemented this model for my research, but recently i start to use Pytorch.
So i would not update this implementation (only maintenance).

Also, we can refer FAIR's awesome open source library for vqa: [Pythia](https://learnpythia.readthedocs.io/en/latest/tutorials/quickstart.html)

**Comment by Nadian-Ali** [comment]
Thank you for your helpful information.  I hope i dont have to switch to
pytorch
